sample/000-dogs-app/src/health.module.ts

```diff
@@ -1,20 +1,23 @@
 import { Module } from '@nestjs/common';
+import { MongooseModule } from '@nestjs/mongoose';
 import {
+  DNSHealthIndicator,
+  MongooseHealthIndicator,
   TerminusModule,
   TerminusModuleOptions,
-  DNSHealthIndicator,
 } from '../../../lib';
-
-import { DogModule } from './dog/dog.module';
+import { CatHealthIndicator } from './cat/cat.health';
 import { CatModule } from './cat/cat.module';
 
 import { DogHealthIndicator } from './dog/dog.health';
-import { CatHealthIndicator } from './cat/cat.health';
+
+import { DogModule } from './dog/dog.module';
 
 const getTerminusOptions = (
   dogHealthIndicator: DogHealthIndicator,
   catHealthIndicator: CatHealthIndicator,
   dnsHealthIndicator: DNSHealthIndicator,
+  mongoHealthIndicator: MongooseHealthIndicator,
```

Comment:
We should keep the sample dogs sample app as it is. It is useful for the user to start the sample app, without having to start a MongoDB instance locally.
If you want to create a sample, feel free to create a new folder `sample/001-mongoose`.

Reply:
Done